libs/cache_line/src/lib.rs

```diff
@@ -0,0 +1,151 @@
+#![feature(const_trait_impl)]
```

Comment:
Add a small crate-level doc comment here, can mostly just link to the main array type.